Add queue command User Guide (#138)
CS2103-AY1819S1-W14-1/main: docs/UserGuide.adoc

`selectpatient 1` +
Selects the 1st patient in the results of the `find` command.

=== Assign a patient into the queue : `enqueuepatient`

Assigns the patient based on the index number used in the displayed patient list. +
Format: `enqueuepatient INDEX`

****
* Assigns the patient into the queue (first in first out manner)
* The index refers to the index number shown in the displayed patient list.
* The index *must be a positive integer* `1, 2, 3, ...`
****

> Examples:

Reviewer: ```suggestion
The command can also be used in combination with other commands.
Examples:
```

* `list` +
`enqueuepatient 7` +

> Selects the 7th patient in the displayed list of patients resulted from the `list` command

Reviewer: ```suggestion
Selects the 7th patient in the displayed list of patients resulting from the `list` command.
```

* `find Logan` +
`enqueuepatient 1` +

> Selects the 1st patient in the displayed list of patients whose name contain *Logan* (case insensitive) resulted from the `find Logan` command

Reviewer: ```suggestion
Selects the 1st patient in the displayed list of patients whose name contains *Logan* (case insensitive) resulting from the `find Logan` command.
```

Author: Noted with thanks
